Examples/Sample_xDscWebServiceRegistrationWithSecurityBestPractices.ps1

```diff
@@ -53,7 +53,7 @@ configuration Sample_xDscWebServiceRegistrationWithEnhancedSecurity
             DependsOn               = "[WindowsFeature]DSCServiceFeature" 
             RegistrationKeyPath     = "$env:PROGRAMFILES\WindowsPowerShell\DscService"   
             AcceptSelfSignedCertificates = $true
-            UseUpToDateSecuritySettings = $true
+            UseSecurityBestPractices = $true
```

Comment:
>UseSecurityBestPractices = $true [](start = 12, length = 32)

One of my concerns was that once the user sets this to true, the settings will remain enabled even if the user sets this to $false.

I think most users will see "SecurityBestPractices" and set this to $true. If this breaks their environment, do they have a way to disable the settings?